edge-e2e/wrapper/nodejs-server-server/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "iot-sdk-device-client-rest-api",
+  "version": "1.0.0",
+  "description": "REST API definition for End-to-end testing of the Azure IoT SDKs.  All SDK APIs that are tested by our E2E tests need to be defined in this file.  This file takes some liberties with the API definitions.  In particular, response schemas are undefined, and error responses are also undefined.",
+  "main": "index.js",
+  "scripts": {
+    "prestart": "npm install",
+    "start": "node index.js"
+  },
+  "keywords": [
+    "swagger"
+  ],
+  "license": "Unlicense",
+  "private": true,
+  "dependencies": {
+    "async": "^2.6.1",
+    "azure-event-hubs": "0.0.8",
+    "azure-iot-common": "1.7.1",
+    "azure-iot-device": "1.7.1",
+    "azure-iot-device-amqp": "1.7.1",
+    "azure-iot-device-http": "1.7.1",
+    "azure-iot-device-mqtt": "1.7.1",
+    "azure-iothub": "1.7.1",
```

Comment:
needs to be caret references to latest SDK packages

Reply:
OK, but we don't do that anywhere else.  Is this something new that we're able to do because we have Lerna?

---
In reply to: [210696479](https://github.com/Azure/azure-iot-sdk-node/pull/346#discussion_r210696479) [](ancestors = 210696479)

Reply:
no, the only reason we don't do it is because packages are linked together - as a user of our SDK i should be able to rely on using the latest version, that's why i'm suggesting this: because users will do that and therefore this may catch bugs.
if you end up linking everything with lerna then I don't think it matters


---
In reply to: [212857354](https://github.com/Azure/azure-iot-sdk-node/pull/346#discussion_r212857354) [](ancestors = 212857354,210696479)